tests/test_feature.rb

```diff
@@ -41,7 +41,7 @@ def vdc_lc_state(type=nil)
   # feature test helper
   def feature(feat)
     # Get the feature name string from the yaml
-    ref = cmd_ref.lookup('feature', feat).to_s[/set_value: feature (.*)/]
+    ref = cmd_ref.lookup('feature', feat).to_s[/set_value:.*feature (.*)/]
```

Comment:
Changed my mind on this.  I believe that `.*` is the best option here instead of trying to optionally match `<state>`.